lib/matplotlib/font_manager.py

```diff
@@ -947,6 +948,24 @@ def copy(self):
         return new
 
 
+class FontsInterface:
```

Comment:
Here would be a great place to add a docstring Assume the reader doesn't know what this class is for - "FontsInterface" is quite generic. It doesn't have to be long, just a couple sentences about why there is this class and what you could do with it.

Reply:
Right, I'll add it in the next commits 👍🏼

Reply:
^^same as above